tools/sigh.ts

```diff
@@ -89,6 +89,10 @@ const steps: {[index: string]: ((args?: string[]) => boolean|Promise<boolean>)[]
   devServer: [peg, build, webpack, devServer],
   languageServer: [peg, build, buildLS, webpackLS],
   run: [peg, build, runNodeScript],
+  ts: [
```

Comment:
It's not clear what this command is for, exactly. `ts` obviously invokes TypeScript, but it seems unexpected for that to include steps like `devServerAsync` and `testWdioShells`. Could you use a more descriptive command name, please?